packages/consensus/fcob/storage.go

```diff
@@ -580,6 +583,22 @@ func (t *TimestampOpinion) ObjectStorageValue() []byte {
 		Bytes()
 }
 
+// SetLiked sets the opinion's liked.
+func (t *TimestampOpinion) SetLiked(newOpinion opinion.Opinion) {
```

Comment:
This should not go into FCoB because it is not FCoB related

Reply:
I have it all refactored on a separate branch, to make transaction and message opinions be similar in terms of implemention.

Reply:
ok